roles/directory/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+- name: Configure reverseproxy
+  block:
```

Comment:
I copied this block thingy from roles/mattermost/tasks/main.yml but the Ansible output then looks really confusing:
```
PLAY [servers] *************************************************************************************

The name of the block doesn't appear anywhere and the sub-tasks just get named `copy` and `systemd`
TASK [Gathering Facts] *****************************************************************************
ok: [katharina.chaospott.de]

TASK [directory : copy] ****************************************************************************
--- before
+++ after: /home/roughl/projects/SpaceAPI/provisioning/roles/directory/files/directory.spaceapi.net
@@ -0,0 +1,4 @@
+directory.spaceapi.net {
+  redir / https://spaceapi.fixme.ch/directory.json 307
+}
+

changed: [katharina.chaospott.de]

TASK [directory : systemd] *************************************************************************
changed: [katharina.chaospott.de]
```

Reply:
I'll refactor this later (and also use notify to restart caddy) so I'll merge it as is.